gdxsv_patch_langtexture/gen_translation.py

```diff
@@ -0,0 +1,45 @@
+import csv
+import sys
+import codecs
+
+with codecs.open("../core/gdxsv/gdxsv_translation_patch.inc","w",encoding="shift_jis_2004") as f:
+    sys.stdout = f
+    print("""
+//
+//  gdxsv_translation_patch.inc
+//  gdxsv
+//
+//  Created by Edward Li on 3/6/2021.
+//  Copyright 2022 gdxsv. All rights reserved.
+//
+#define TRANSLATE(offset,length,original,cantonese,english) TranslationWithMaxLength<length>(offset,original,cantonese,english)
+#define CUSTOMIZE(offset,length,original,cantonese,english,japanese) TranslationWithMaxLength<length>(offset,original,cantonese,english,japanese)
+
+const u32 offset = 0x8C000000 + 0x00010000;
+
+Translation translations[] = {
```

Comment:
It is better to make it static since it will be evaluated many times at runtime.

Reply:
done